src/Components/missions/Missions.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { toggleJoining } from '../../redux/Missions/missionsReducer';
+import style from './missions.module.css';
+
+function Missions() {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <table className={style.table} data-testid="list-of-missions">
+        <thead className={style.thead}>
+          <tr className={style.header}>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        {missions.map((el) => (
+          <tbody key={el.id} className={style.item}>
+            <td className={style.row}>{el.mission_name}</td>
+            <p className={style.description}>{el.description}</p>
+            <td>
+              <button type="submit" className={el.joined ? style.inactiveMember : style.activeMember}>
+                {
+                el.joined ? 'NOT A MEMBER' : 'Active Member'
```

Comment:
- [ ] Kindly change the texts. The user becomes an "Active Member" after joining the mission. And "Not a member" when leaving the mission.